bindings/rust/evmc-client/src/types.rs

```diff
@@ -0,0 +1,18 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2019 The EVMC Authors.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pub use evmc_sys as ffi;
+
+pub const ADDRESS_LENGTH: usize = 20;
+pub const BYTES32_LENGTH: usize = 32;
+pub type Address = [u8; ADDRESS_LENGTH];
+pub type Bytes32 = [u8; BYTES32_LENGTH];
+pub type Bytes = [u8];
+
+pub type StorageStatus = ffi::evmc_storage_status;
+pub type CallKind = ffi::evmc_call_kind;
+pub type Revision = ffi::evmc_revision;
+pub type StatusCode = ffi::evmc_status_code;
+pub type Capabilities = ffi::evmc_capabilities;
```

Comment:
Why not use `evmc-vm::types` instead of duplicating? We could later refactor to pull them out somewhere.

Reply:
Yes, there is needless. I don't notice main line is already added alias types after I fork it. 
Fixed in 7dbcfe0